plugins/woocommerce/src/Blocks/InteractivityComponents/CheckboxList.php

```diff
@@ -24,12 +24,9 @@ public static function render( $props ) {
 		wp_enqueue_script( 'wc-interactivity-checkbox-list' );
 		wp_enqueue_style( 'wc-interactivity-checkbox-list' );
 
-		$items = $props['items'] ?? array();
-
+		$items                 = $props['items'] ?? array();
 		$checkbox_list_context = array( 'items' => $items );
-
-		// Items should be an array of objects with a label (which can be plaintext or HTML) and value property.
-		$items = $props['items'] ?? array();
+		$on_change             = $props['on_change'] ?? '';
```

Comment:
brought declaration of props variables up earlier in file instead of referencing property later in the template. I just think it makes it easier for users of the component to see what props are declared and available.